modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Drains nodes in singular order based on the selected machine config pool.
+* Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+* Restarts the node with the modified override file.
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Convert the rule to Base64 encoded by running the following command:
++
+[source,terminal]
+----
+$ base64 /path/to/file/
+----
+
+. Complete one of the following steps to define a secondary interface for your network. Ensure that the secondary interface is not bound to the `br-ex` bridge.
++
+[IMPORTANT]
```

Comment:
Important block is good, I would also indicate that the interface cannot be already used by `br-ex` as well.

~~~
Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface. Ensure that the secondary interface or sub-interfaces (if defining a bond) are not already utilized by existing `br-ex` OVN-kubernetes network deployment.
~~~

Reply:
:heavy_check_mark: